lib/vmdb/loggers/container_logger.rb

```diff
@@ -0,0 +1,53 @@
+module Vmdb::Loggers
+  class ContainerLogger < VMDBLogger
+    def initialize(logdev = STDOUT, *args)
+      super
+      self.level = DEBUG
+      self.formatter = Formatter.new
+    end
+
+    def level=(_new_level)
+      super(DEBUG) # We want everything written to the ContainerLogger written to STDOUT
+    end
+
+    def filename
+      "STDOUT"
+    end
+
+    class Formatter < VMDBLogger::Formatter
+      SEVERITY_MAP = {
+        "DEBUG"   => "debug",
+        "INFO"    => "info",
+        "WARN"    => "warning",
+        "ERROR"   => "err",
+        "FATAL"   => "crit",
+        "UNKNOWN" => "unknown"
+        # Others that don't match up: alert emerg notice trace
+      }.freeze
+
+      def call(severity, time, progname, msg)
+        # From https://github.com/ViaQ/elasticsearch-templates/releases Downloads asciidoc
```

Comment:
Follow the link, under "Downloads" you'll see an asciidoc which contains the "schema"